opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -213,11 +225,19 @@ const CsvMapperForm: FunctionComponent<CsvMapperFormProps> = ({
                   <Typography>{t('Semicolon')}</Typography>
                 </div>
               </div>
-              <div className={classes.center} style={{ marginTop: 20 }}>
+              <div className={classes.center}>
+                  <Field
```

Comment:
maybe add after this field an info icon with tooltip like we do with the "my file contains header" checkbox ?

Suggestion:

```
<Tooltip
    title={t(
      'Every line that begins with this character will be skipped during parsing (for example: #).',
    )}
  >
    <InformationOutline
      fontSize="small"
      color="primary"
      style={{ cursor: 'default' }}
    />
  </Tooltip>
```

Reply:
Good idea, will do it.